test/univariates.jl

```diff
@@ -127,13 +127,13 @@ function verify_and_test(D::Union{Type,Function}, d::UnivariateDistribution, dct
 
     try
         m = mgf(d,0.0)
-        @test m == 1.0
+        @test m ≈ 1.0 atol=10*eps(Float64) rtol=0
```

Comment:
I think with the `expm1` suggestions above this could be reverted.
```suggestion
        @test m == 1.0
```